engine/apps/slack/scenarios/notification_delivery.py

```diff
@@ -52,18 +49,14 @@ def process_signal(self, log_record):
                         self.post_message_to_channel(
                             f"Failed to send an SMS to {user_verbal_with_mention}. Phone number is not verified",
                             alert_group.slack_message.channel_id,
-                            color="red",
                         )
                     elif log_record.notification_channel == UserNotificationPolicy.NotificationChannel.PHONE_CALL:
                         self.post_message_to_channel(
                             f"Failed to call to {user_verbal_with_mention}. Phone number is not verified",
                             alert_group.slack_message.channel_id,
-                            color="red",
                         )
 
-    def post_message_to_channel(self, text, channel, color=None, footer=None):
-        # TODO: No color in blocks, use prefix emoji?
-        # color_id = self.get_color_id(color)
```

Comment:
I think we can also delete the `get_color_id` method, LGTM otherwise

Reply:
I'll pass through and cleanup in another PR, let's get this deployed.